packages/react-scripts/template/src/utils/is-promise.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Right, it seems that fire-action also is not used anywhere. I thought we had this because we wanted it to be there. Should I drop everything that is not used?